config/locales/en.yml

```diff
@@ -1707,6 +1707,25 @@ en:
             The FileYourStateTaxes team
           subject: Remember to finish your state tax return
         sms: "Hi %{primary_first_name} - You started a state tax return with FileYourStateTaxes but have not finished. \n\nGo to http://www.fileyourstatetaxes.org/en/login-options to finish them today. The last day to submit your state tax return is October 22.\n\nIf you have already filed using another service, no action is required.\n\nNeed help? Email us at help@fileyourstatetaxes.org\n\nBest,\nThe FileYourStateTaxes team\n"
+      october_transfer_reminder:
+        email:
+          body: |
+            Hi!
+
+            It looks like you created an account with FileYourStateTaxes but haven't transferred your federal return information from IRS Direct File. If you want to file your state return with us please do so before October 22nd.
```

Comment:
on em’s ticket FileYourStatTaxes is a link: https://codeforamerica.atlassian.net/browse/FYST-2263

i copied that pattern on mine as well. you can also double check with Erica.